cloud/deploys/dev_azure/civiform_config.example.sh

```diff
@@ -9,6 +9,7 @@ export CIVIC_ENTITY_FULL_NAME="City of Sooschester"
 export CIVIC_ENTITY_SUPPORT_EMAIL_ADDRESS="civiform-azure-staging-email@googlegroups.com"
 export CIVIC_ENTITY_LOGO_WITH_NAME_URL="https://www.cityofrochester.gov/images/header-logo.png"
 export CIVIC_ENTITY_SMALL_LOGO_URL="https://www.cityofrochester.gov/assets/0/117/8589934986/c6791498-270e-40b1-9b81-aca8a510c1b5.jpg"
+export CIVIFORM_TIME_ZONE_ID="America/Los_Angeles"
```

Comment:
You'll need to make this change in the https://github.com/civiform/civiform-deploy repo, too

Reply:
Thanks for flagging, I'll send out a PR for that today. Since we marked this as "optional", nothing will break in the civiform-deploy repo without this change. However, without making it there, it won't be very discoverable at all :).